backend/utils/llm.py

```diff
@@ -1,12 +1,32 @@
 import os
 from dotenv import load_dotenv
 from mistralai.client import MistralClient
+from mistralai.models.chat_completion import ChatMessage
 
 load_dotenv()
 
-endpoint = os.environ["INFER_GPT_MISTRAL_LARGE_ENDPOINT"]
-api_key = os.environ["INFER_GPT_MISTRAL_LARGE_KEY"]
-model = "azureai"
+try:
+  endpoint = os.environ["INFER_GPT_MISTRAL_LARGE_ENDPOINT"]
+  api_key = os.environ["INFER_GPT_MISTRAL_LARGE_KEY"]
+  key = os.getenv("OPENAI_API_KEY")
```

Comment:
Use both `os.getenv` and `os.environ`. Should stick to one for consistency. Don't believe it makes a difference which one.